tests/test_schema.py

```diff
@@ -0,0 +1,49 @@
+import json
+
+from conda_smithy.schema import ConfigModel
+from conda_smithy.validate_schema import (
+    CONDA_FORGE_YAML_SCHEMA_FILE,
+    validate_json_schema,
+)
+
+
+def test_schema_up_to_date():
+    model = ConfigModel()
+
+    json_blob_from_model = (
+        json.dumps(model.model_json_schema(), indent=2) + "\n"
+    )
+    assert CONDA_FORGE_YAML_SCHEMA_FILE.exists(), (
+        "The config schema file does not exist. "
+        "Run `python -m conda_smithy.schema` to generate it."
+    )
+    json_blob_from_code = CONDA_FORGE_YAML_SCHEMA_FILE.read_text(
+        encoding="utf-8"
+    )
+    assert json.loads(json_blob_from_model) == json.loads(
```

Comment:
`json.loads(json_blob_from_model)` essentially does `json.loads(json.dumps(model.model_json_schema(), indent=2) + "\n")`
Why is there a round trip here?

Reply:
Oh sorry. That was to make the diff in the tests easier to read for debugging. I'm happy to fix in another pr if this matters to you. Let me know what you want done.